README.md

```diff
@@ -1,29 +1,25 @@
-Semantic-UI-Angular
-===================
+## SemanticUI-Angular [![Build Status](https://travis-ci.org/ClickerMonkey/SemanticUI-Angular.svg?branch=master)](https://travis-ci.org/ClickerMonkey/SemanticUI-Angular) [![devDependency Status](https://david-dm.org/ClickerMonkey/SemanticUI-Angular/dev-status.svg?branch=master)](https://david-dm.org/ClickerMonkey/SemanticUI-Angular#info=devDependencies)
```

Comment:
Keep the original heading format, looks better. Need point to the semantic-org account/repo